<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOkvedDialog</class>
 <widget class="QDialog" name="AddOkvedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить новую позицию</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>561</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Раздел</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QRadioButton" name="oldRazdelButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Существующий</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="newRazdelButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Новый</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="comboRazdels"/>
        </item>
        <item>
         <widget class="QLineEdit" name="newRazdelEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddOkvedDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddOkvedDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oldRazdelButton</sender>
   <signal>clicked()</signal>
   <receiver>AddOkvedDialog</receiver>
   <slot>changeRazdelButtonsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newRazdelButton</sender>
   <signal>clicked()</signal>
   <receiver>AddOkvedDialog</receiver>
   <slot>changeRazdelButtonsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeRazdelButtonsClicked()</slot>
 </slots>
</ui>
